### PR TITLE
feat: keep3r_v2 migration

### PR DESCRIPTION
**Pull request by m-r-g-t**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by m-r-g-t** [comment]
I need some assistant to detect what is the problem on the sql.

error message it is not very clear,

22:07:49  Database Error in model keep3r_network_addLiquidityToJob (models/keep3r_network/keep3r_network_addLiquidityToJob.sql)
22:07:49    TrinoUserError(type=USER_ERROR, name=SYNTAX_ERROR, message="line 26:9: mismatched input '{'. Expecting: '(', 'JSON_TABLE', 'LATERAL', 'TABLE', 'UNNEST', <identifier>", query_id=20230911_220749_19553_caqi8)

could you please give me a hand or point me in the correct direction?

File: models/keep3r_network/keep3r_network_addLiquidityToJob.sql [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{{ config(
+      tags=['dunesql']
+      , alias = alias('etv_addliquiditytoJob')
```
Comment (jeff-dude) [inline models/keep3r_network/keep3r_network_addLiquidityToJob.sql:3]: we universally use `lower_case_with_underscore` for naming standard, for files/directories/aliases/schemas/columns, can you ensure all here is updated to reflect that?

File: models/keep3r_network/keep3r_network_addLiquidityToJob.sql [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    UNION ALL
+    SELECT
+      call_block_time as block_time,
+      CAST(_amount as double) / 1e18 as amount,
+      _job as job,
+      _liquidity as liquidity,
+      'addLiquidityToJob' as event,
+      'ethereum' as blockchain
+    FROM
+        { source(
```
Comment (jeff-dude) [inline models/keep3r_network/keep3r_network_addLiquidityToJob.sql:33]: you are missing a `{` here, which is why you see the error. jinja syntax requires two of them, like your example on line 18

File: models/keep3r_network/keep3r_network_addLiquidityToJob.sql [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    UNION ALL
+    SELECT
+      call_block_time as block_time,
+      CAST(_amount as double) / 1e18 as amount,
+      _job as job,
+      _liquidity as liquidity,
+      'addLiquidityToJob' as event,
+      'optimism' as blockchain
+    FROM
+        { source(
```
Comment (jeff-dude) [inline models/keep3r_network/keep3r_network_addLiquidityToJob.sql:48]: same here

File: models/keep3r_network/keep3r_network_addLiquidityToJob.sql [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    UNION ALL
+    SELECT
+      call_block_time as block_time,
+      CAST(_amount as double) / 1e18 as amount,
+      _job as job,
+      _liquidity as liquidity,
+      'addLiquidityToJob' as event,
+      'polygon' as blockchain
+    FROM
+        { source(
```
Comment (jeff-dude) [inline models/keep3r_network/keep3r_network_addLiquidityToJob.sql:63]: same here

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/keep3r_network/keep3r_network_token_price_legacy.sql [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{{ config( 
+  schema = 'keep3r_network',
+  alias = alias('token_price', legacy_model=True),
```
Comment (jeff-dude) [inline models/keep3r_network/keep3r_network_token_price_legacy.sql:3]: minor: looks like dummy legacy file here is missing `s` at the end. dunesql version uses `token_prices`
this applies to both alias here and file name

File: dbt_project.yml [hunk 6]
```diff
@@ -827,9 +827,7 @@ models:
         +schema: ironbank_optimism
 
     keep3r_network:
-      +schema: keep3r_network
-      ethereum:
-        +schema: keep3r_network_ethereum
+      +schema: keep3r_network
```
Comment (jeff-dude) [inline dbt_project.yml:830]: can you remove lines 829/830 altogether, then add schema property to each model config block next to alias instead?

we're moving towards a cleaner `dbt_project` file in the future, with removing schemas being a big part. easier to keep in the model itself for quick reference to full table name

File: models/keep3r_network/keep3r_network_job_migration.sql [hunk 7]
```diff
@@ -0,0 +1,66 @@
+{{ config (
+    tags=['dunesql']
+    , alias = alias('etv_job_migration')
+    , unique_key = ['blockchain', 'block_time']
```
Comment (jeff-dude) [inline models/keep3r_network/keep3r_network_job_migration.sql:4]: minor: unique key configs on all spells in this PR are not really needed, as they are views. most important for incremental spells.

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Pull request merged**